builtin/providers/rundeck/provider.go

```diff
@@ -0,0 +1,50 @@
+package rundeck
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/apparentlymart/go-rundeck-api/rundeck"
+)
+
+func Provider() terraform.ResourceProvider {
+	return &schema.Provider{
+		Schema: map[string]*schema.Schema{
+			"url": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "URL of the root of the target Rundeck server.",
+			},
```

Comment:
I guess not... my own personal style is to put the stuff that talks about the user in the env vars and the stuff that talks about the infrastructure in the config, but no reason to force my style on everyone.

(My motivation: usually in my world the targets of these things come from other resources like ``consul_keys`` and ``terraform_remote_state``, rather than being set directly by the user.)